onboard/catkin_ws/src/offboard_comms/Arduino Sketchbooks/ThrusterArduino/ThrusterArduino.ino

```diff
@@ -63,6 +65,10 @@ void setup()
         thrusters[i].initialize(&pwm_multiplexer);
         thrusters[i].attach(i);
     }
+
+    // Write the 1500 PWM to all thrusters to stop them
+    for (uint8_t i = 0; i < NUM_THRUSTERS; ++i)
+        thrusters[i].write(THRUSTER_STOP_PWM + THRUSTER_PWM_OFFSET);
```

Comment:
@mbryant2025 I don't think this loop is necessary as `THRUSTER_STOP_PWM + THRUSTER_PWM_OFFSET` will be written to all thrusters in `loop()`.